next.config.mjs

```diff
@@ -53,6 +47,48 @@ const nextConfig = {
         port: '',
         pathname: '/**',
       },
+      {
+        protocol: 'https',
+        hostname: 'raw.githubusercontent.com', // TODO: this one can be dangerous
```

Comment:
to remove when we move on prod

Reply:
We unfortunately cannot because of some tokens image that we get from this (mainly trustwallet)